docs/installation.rst

```diff
@@ -148,7 +148,7 @@ Python-based prerequisites
     The following outlines installing Keylime under the Python 2 environment,
     work is underway to port Keylime to Python 3.
 
-The following python packages are required:
+The following Python packages are required:
 
 * pycryptodomex>=3.4.1
```

Comment:
Change looks good , just noticed we need to update these dependencies (we dropped pycryptodome for python-cryptography) - you can fix here if you like or we can do a follow up PR.

Reply:
Let's do it here. If you give the min. version for python-cryptography i will update my PR.

Reply:
Just saw we still had this hanging while doing some tidying up. Let's try and close it! @lukehinds @mpeters do you now the min version for `python-cryptography`?

Reply:
This was now bumped to v3.3.2 for some security fixes and dependency problems.